common/k8s/src/middleware/metadata.rs

```diff
@@ -205,6 +205,7 @@ impl Middleware for K8sMetadata {
         });
     }
 
+    #[allow(clippy::question_mark)]
```

Comment:
Not sure why clippy is now complaining about this block of code but it occurs both locally and on Jenkins so I added the allow tag.